Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -697,6 +703,12 @@ public final class PackageBuilder {
         )
         let (sources, resources) = try sourcesBuilder.run()
 
+        // Make sure developmentRegion is set if the target has localized resources.
+        let hasLocalizedResources = resources.contains(where: { $0.localization != nil })
+        if hasLocalizedResources && manifest.developmentRegion == nil {
```

Comment:
This seems like something we should diagnose during manifest parsing, right?

Reply:
How would you suggest doing it during manifest parsing? It would require doing a lot of the logic that we currently do in `TargetSourcesBuilder`: search .process paths for lproj directories or for .process paths defined with an explicit localization. It's doable, but it repeats some of the same logic in a different place.

Reply:
aha, then never mind. I thought we can compute it without accessing the target's sources.